paper.tex

```diff
@@ -160,14 +161,14 @@ \section*{Ancestral graphs and stochastic processes}
 Each lineage consists of a list of
 disjoint ancestry segments $(\ell, r, a)$, where
 $[\ell, r)$ is a half-closed genomic interval and $a$ is an integer
-tracking the number of samples the lineage is ancestral to over the interval.
+tracking the number of samples to which the lineage is ancestral over that interval.
 (We also usually track the tree node associated with each segment, but
 that is not important for our purposes here so we omit it.)
 If we have $n$ samples and genome of length $m$, the process begins with $n$ lineages
 of the form $[(0, m, 1)]$. The process then works backwards in time from
 the present day as a series of random common ancestor or recombination events.
-Recombination events occur at a rate determined by the amount and spatial
-distribution of ancestral material among lineages.
+Recombination events occur at a rate determined by the genetic location and amount
```

Comment:
As Jere suggests, "at a rate determined by the amount of ancestral material and the way in which it is distributed along a chromosome"

"Genetic location" suggests we're talking about genetic maps or something, which is just an extra layer of confusion to avoid. (I mean, we are, but it's not important)